app/src/main/res/layout/mark_chapters_completed_fragment.xml

```diff
@@ -114,6 +115,7 @@
         android:layout_height="match_parent"
         android:background="@color/profileEditBackground"
         android:clipToPadding="false"
+        android:contentDescription="@string/mark_chapters_completed_recycler_view"
```

Comment:
Why do we need this? `RecyclerView` should never have a content description.

Reply:
PTAL at https://github.com/oppia/oppia-android/pull/3470#discussion_r677278179

Reply:
This error is because of when this test is running there is no content inside recyclerview.

Instead of doing this solution, try to update the test case such that there is content inside recyclerview and this error won't appear in that case.

@yashraj-01

Reply:
![Screenshot from 2021-07-29 18-25-36](https://user-images.githubusercontent.com/63469151/127496127-9b979621-3367-4d55-b4c1-0c55dcb79525.png)
@rt4914 it seems like this was an issue with the emulator as when I ran the Espresso tests on a physical device, all the tests passed successfully.

Reply:
So, I have removed the `contentDescription` from the recyclerview.

Reply:
Tests are passing.